x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/create_layout.ts

```diff
@@ -16,9 +16,9 @@ interface LayoutParams {
 
 export function createLayout(server: KbnServer, layoutParams: LayoutParams): Layout {
   if (layoutParams && layoutParams.id === LayoutTypes.PRESERVE_LAYOUT) {
-    return new PreserveLayout(layoutParams.id, layoutParams.dimensions);
+    return new PreserveLayout(layoutParams.dimensions);
```

Comment:
Add the typescript change above, since layoutParams might be null: `layoutParams?: LayoutParams`